### PR TITLE
Introduce OpenSUSE Tumbleweed image

### PR DESCRIPTION
**Pull request by jelly**
* [x] image-refresh opensuse-tumbleweed

### CODE REVIEW

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-opensuse-tumbleweed-20240304-161009-3bddbb0b-image-refresh-opensuse-tumbleweed/log.html

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-opensuse-tumbleweed-52d111b3-20240308-082241/log.html

**Comment by cockpituous** [comment]
image-refresh opensuse-tumbleweed done: https://github.com/cockpit-project/bots/commits/image-refresh-opensuse-tumbleweed-20240320-154432

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-opensuse-tumbleweed-c05d91e4-20240320-153113/log.html

**Comment by martinpitt** [comment]
Worked! Please cherry-pick https://github.com/cockpit-project/bots/commit/10c69bb74747c66af71250fea5be3fad684fda1e then we can do the next step and run a simple test against it. We usually start with `starter-kit`, as tests are trivial, and we can make sure that package building, installation, and starting cockpit works in general. But of course we'll leave the strategy to you.

File: images/scripts/lib/build-deps.sh [hunk 1]
```diff
@@ -23,7 +23,11 @@ case "$OS_VER" in
     rhel*8|centos*8)
         spec=$($GET "$COCKPIT_GIT/rhel-8/tools/cockpit.spec")
         ;;
-
+    *suse*)
+        # macro for detmerning suse version is %suse_version
```
Comment (martinpitt) [inline images/scripts/lib/build-deps.sh:27]: Typo

File: images/scripts/opensuse-tumbleweed.setup [hunk 2]
```diff
@@ -0,0 +1 @@
+opensuse.setup
```
Comment (martinpitt) [inline images/scripts/opensuse-tumbleweed.setup:1]: Let's not do this indirection just yet -- if we get a second similar suse image, we can do the factorization then. The micro image setup will most likely look completely different.

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by cockpituous** [comment]
image-refresh opensuse-tumbleweed done: https://github.com/cockpit-project/bots/commits/image-refresh-opensuse-tumbleweed-20240323-055237

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-opensuse-tumbleweed-be5a86cc-20240323-053844/log.html

**Pull request closed without merging**